### PR TITLE
Demo

### PR DESCRIPTION
**Pull request by mthava**
# Description

Fix App - insight / Log back incompatibilty 

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:

* Hardware:
* O/S and version:
* JDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: classes/production/div-evidence-management-client-api/application-demo.properties [hunk 1]
```diff
@@ -0,0 +1,16 @@
+logging.level.org.springframework.web=DEBUG
+logging.level.uk.gov.hmcts.ccd=DEBUG
+
+#auth.idam.client.baseUrl=http://betaDevBccidamAppLB.reform.hmcts.net
+auth.idam.client.baseUrl=${IDAM_URL:http://localhost:4501}
+#idam.s2s-auth.url=http://betaDevBccidamS2SLB.reform.hmcts.net:80
+idam.s2s-auth.url=${AUTH_PROVIDER_SERVICE_CLIENT_BASEURL:http://localhost:4502}
+#idam.auth.secret=RJBNLKOV7OJR52VX
```
Comment (tchow8) [inline classes/production/div-evidence-management-client-api/application-demo.properties]: Please remove secret

File: classes/production/div-evidence-management-client-api/application.properties [hunk 2]
```diff
@@ -0,0 +1,21 @@
+logging.level.org.springframework.web=DEBUG
+logging.level.uk.gov.hmcts.ccd=DEBUG
+
+env=${CNP_ENVIRONMENT:local}
+
+#auth.idam.client.baseUrl=http://betaDevBccidamAppLB.reform.hmcts.net
+auth.idam.client.baseUrl=http://localhost:4501
+#idam.s2s-auth.url=http://betaDevBccidamS2SLB.reform.hmcts.net:80
+idam.s2s-auth.url=http://localhost:4502
+#idam.auth.secret=RJBNLKOV7OJR52VX
```
Comment (tchow8) [inline classes/production/div-evidence-management-client-api/application.properties]: Please remove secret before merge

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -16,6 +16,11 @@ evidence.management.upload.file.url=${EVIDENCE_MANAGEMENT_UPLOAD_FILE_URL:http:/
 evidence.management.store.upload.file.url=${DOCUMENT_MANAGEMENT_STORE_URL:http://localhost:4603}/documents
 
 evidence.management.store.health.url=${DOCUMENT_MANAGEMENT_STORE_URL:http://localhost:3405}/health
+#evidence.management.store.health.url="https://dm-store-app-demo.service.core-compute-demo.internal/health"
```
Comment (Rob-Doherty) [inline src/main/resources/application.properties]: can this comment be removed?

File: classes/test/div-evidence-management-client-api/fileuploadresponse.txt [hunk 4]
```diff
@@ -0,0 +1,46 @@
+{
```
Comment (Rob-Doherty) [inline classes/test/div-evidence-management-client-api/fileuploadresponse.txt]: theres quite a few new test files / test data, but not any obvious new test to use them. Has that been forgotten?

File: classes/production/div-evidence-management-client-api/application-aat.properties [hunk 5]
```diff
@@ -0,0 +1,13 @@
+logging.level.org.springframework.web=DEBUG
+logging.level.uk.gov.hmcts.ccd=DEBUG
+
+auth.idam.client.baseUrl=http://betaDevBccidamAppLB.reform.hmcts.net
```
Comment (Rob-Doherty) [inline classes/production/div-evidence-management-client-api/application-aat.properties]: AAT is not pointing to DevB tactical, it points to preprod

File: classes/production/div-evidence-management-client-api/application-local.properties [hunk 6]
```diff
@@ -0,0 +1,14 @@
+logging.level.org.springframework.web=DEBUG
+logging.level.uk.gov.hmcts.ccd=DEBUG
+
+auth.idam.client.baseUrl=http://localhost:4501}
```
Comment (Rob-Doherty) [inline classes/production/div-evidence-management-client-api/application-local.properties]: there is a missing `{` here

Comment (Rob-Doherty) [reply classes/production/div-evidence-management-client-api/application-local.properties]: if it didn't fail anything, is the line actually needed?

**Comment by tchow8** [comment]
Looks like the codebase got polluted. Resetting this branch to master and applying the changes.

File: classes/test/div-evidence-management-client-api/fileuploadresponse.txt (hunk 4 above)
Comment (mthava) [reply classes/test/div-evidence-management-client-api/fileuploadresponse.txt]: Guys these are classes folder. Something has gone wrong with git ignore file. Just remove this files and add git ignore

**Pull request closed without merging**